commit fa92c4e6b244ae5814ac1ad87d00cf40b6d7199f
Author: David Abdurachmanov <david.abdurachmanov@gmail.com>
Date:   Thu Dec 6 16:26:34 2018 +0100

    riscv: fix trace_sys_exit hook
    
    [ Upstream commit 775800b0f1d7303d4fd8ce0e0d9eca4ff2f338f2 ]
    
    Fix compilation error.
    
    Signed-off-by: David Abdurachmanov <david.abdurachmanov@gmail.com>
    Signed-off-by: Palmer Dabbelt <palmer@sifive.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/riscv/kernel/ptrace.c b/arch/riscv/kernel/ptrace.c
index 9f82a7e34c64..9db7d0076375 100644
--- a/arch/riscv/kernel/ptrace.c
+++ b/arch/riscv/kernel/ptrace.c
@@ -120,6 +120,6 @@ void do_syscall_trace_exit(struct pt_regs *regs)
 
 #ifdef CONFIG_HAVE_SYSCALL_TRACEPOINTS
 	if (test_thread_flag(TIF_SYSCALL_TRACEPOINT))
-		trace_sys_exit(regs, regs->regs[0]);
+		trace_sys_exit(regs, regs_return_value(regs));
 #endif
 }
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1380_fa92c4e6b244ae5814ac1ad87d00cf40b6d7199f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1380_fa92c4e6b244ae5814ac1ad87d00cf40b6d7199f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
